src/components/gallery/index.scss

```diff
@@ -89,17 +89,24 @@
     z-index: 20;
 
     @media (min-width: 576px) {
-      width: 450px;
-      height: 310px;
+      width: 448px;
+      height: 308px;
       margin: 0 auto;
     }
 
-    &__icon {
+    &__icon-container {
       position: absolute;
-      top: -30%;
+      top: -25vh;
```

Comment:
Please, do not use negative numbers for css values as possible, refactor this styling approach, thx.

Reply:
Could these changes be made in a later chore branch? since they are not part of the footer functionality of this branch